lint/highlight.py

```diff
@@ -1,193 +0,0 @@
-from collections import defaultdict
-import sublime
-
-from . import persist
-from . import style as style_stores
-from .style import HighlightStyleStore
-from .const import PROTECTED_REGIONS_KEY, WARNING, ERROR, WARN_ERR, INBUILT_ICONS
-
-
-UNDERLINE_FLAGS = sublime.DRAW_NO_FILL | sublime.DRAW_NO_OUTLINE | sublime.DRAW_EMPTY_AS_OVERWRITE
-
-MARK_STYLES = {
-    'outline': sublime.DRAW_NO_FILL,
-    'fill': sublime.DRAW_NO_OUTLINE,
-    'solid_underline': sublime.DRAW_SOLID_UNDERLINE | UNDERLINE_FLAGS,
-    'squiggly_underline': sublime.DRAW_SQUIGGLY_UNDERLINE | UNDERLINE_FLAGS,
-    'stippled_underline': sublime.DRAW_STIPPLED_UNDERLINE | UNDERLINE_FLAGS,
-    'none': sublime.HIDDEN
-}
-
-
-# Dict[view_id, region_keys]
-regions = defaultdict(set)
-
-
-def remember_drawn_regions(view, regions_keys):
-    """Remember draw regions for later clearance."""
-    view_id = view.id()
-    regions[view_id].update(regions_keys)
-
-
-def clear_view(view):
-    """Clear all marks in the given view."""
-    view_id = view.id()
-    region_keys = regions.pop(view_id, [])
-    for key in region_keys:
-        view.erase_regions(key)
-
-
-def get_line_start(view, line):
-    return view.text_point(line, 0)
-
-
-class Highlight:
-    """This class maintains error marks and knows how to draw them."""
-
-    def __init__(self, view):
-        self.view = view
-
-        # Dict[error_type, Dict[style, List[region]]]
-        self.marks = defaultdict(lambda: defaultdict(list))
-        self.style_store = HighlightStyleStore()
```

Comment:
@kaste 
Just a quick note: Did you delete the class declaration as well?
As you deleted its only instantiation, it is not needed anymore.

Reply:
That's your job. 😁

Ahem. No.

Reply:
You kept it. I did not expect `HighlightStyleStore` to escape your wrath.